zerver/tests/test_realm_playgrounds.py

```diff
@@ -0,0 +1,88 @@
+from zerver.lib.test_classes import ZulipTestCase
+from zerver.models import RealmPlayground, get_realm
+
+
+class RealmPlaygroundTests(ZulipTestCase):
+    def test_create_one_playground_entry(self) -> None:
+        iago = self.example_user("iago")
+
+        # Now send a POST request to the API endpoint.
+        payload = {
+            "name": "Python playground",
+            "pygments_language": "Python",
+            "url_prefix": "https://some-website.com/?code="
```

Comment:
Let's use E.g. `python.example.com` for these examples -- that's the RFC example domain.